python/astshim/test.py

```diff
@@ -131,3 +132,66 @@ def checkBasicSimplify(self, amap):
             self.assertEqual(mb.getNout(), cmp3.getNout())
             self.assertEqual(cmp3.getNin(), cmp3simp.getNin())
             self.assertEqual(cmp3.getNout(), cmp3simp.getNout())
+
+
+def makeCoeffs(nIn, nOut):
+    """Make an array of coefficients for astshim.PolyMap for the following equation:
+
+    fj(x) = C0j x0^2 + C1j x1^2 + C2j x2^2 + ... + CNj xN^2
+    where:
+    * i ranges from 0 to N=nIn-1
+    * j ranges from 0 to nOut-1,
+    * Cij = 0.001 (i+j+1)
+    """
+    baseCoeff = 0.001
+    forwardCoeffs = []
+    for out_ind in range(nOut):
+        coeffOffset = baseCoeff * out_ind
+        for in_ind in range(nIn):
+            coeff = baseCoeff * (in_ind + 1) + coeffOffset
+            coeffArr = [coeff, out_ind + 1] + [2 if i == in_ind else 0 for i in range(nIn)]
+            forwardCoeffs.append(coeffArr)
+    return np.array(forwardCoeffs, dtype=float)
+
+
+def makeTwoWayPolyMap(nIn, nOut):
+    """Make an astShim.PolyMap suitable for testing
+
+    The forward transform is as follows:
+    fj(x) = C0j x0^2 + C1j x1^2 + C2j x2^2 + ... + CNj xN^2 where Cij = 0.001 (i+j+1)
+
+    The reverse transform is the same equation with i and j reversed
+    thus it is NOT the inverse of the forward direction,
+    but is something that can be easily evaluated.
+
+    The equation is chosen for the following reasons:
+    - It is well defined for any value of nIn, nOut
```

Comment:
I suggest `all positive values of nIn, nOut` (the important point being to get `positive` in there somehow). I should have done that when I wrote it.

Similarly for the doc of `makeForwardPolyMap` below.